Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -648,15 +648,20 @@ static void registerLogHook()
         return;
 
     os_log_set_hook(OS_LOG_TYPE_DEFAULT, ^(os_log_type_t type, os_log_message_t msg) {
+        if (msg->buffer_sz > 1024)
```

Comment:
Do we need individual length limits if we have this overall limit too?

Reply:
> Do we need individual length limits if we have this overall limit too?

I was thinking it could be good to have a length check in the Networking process too, in case the IPC endpoint is being accessed in other, unexpected ways.

Thanks for reviewing!